lib/bibliothecary/parsers/conda.rb

```diff
@@ -10,18 +10,24 @@ def self.mapping
         {
           match_filename("environment.yml") => {
             parser: :parse_conda_manifest,
-            kind: "manifest"
+            multiple: true,
+            kind: %w[manifest lockfile]
```

Comment:
Could the `multiple` field go away with the change above to instead key off of whether kind is a list?